lib/griddler/sendgrid/adapter.rb

```diff
@@ -49,7 +68,8 @@ def bcc_from_envelope
 
       def charsets
         return {} unless params[:charsets].present?
-        JSON.parse(params[:charsets]).symbolize_keys
+        charsets = JSON.parse(params[:charsets]).symbolize_keys
+        charsets.transform_values!{ "UTF-8" }
```

Comment:
looking at this, it seems potentially destructive, or possibly not i18n-friendly ... 🤔 is there some valid case where someone might want _not_ utf8 charsets?

At this point should we introduce a configuration option, so all this behavior is opt-in?